prod/README.md

```diff
@@ -45,4 +45,15 @@ $ ../tools/download_simulationsoftware.sh
 $ docker build -t simtools-prod .
 ```
 
-Building will take a while and the image is large (2.3 GB).
+Building will take a while and the image is large (~1.3 GB).
+
+**Running**
+
+To run the container that was just built,
+
+```
+docker run --rm -it simtools-prod bash
+```
+
+Notice that in this case we did not include the `external` directory since we did not set it up before. It can be done similarly to the `dev` container if needed though.
```

Comment:
Also think that this section **Running** should be completely removed. There is a running section further up in the readme. I'll make a suggestion.

Reply:
Your suggestion is better for setting up an external directory.

For the running section, I agree it exists but it confused Victor cause it runs the docker image from github rather than the one just built. I will wait for your concrete suggestion before commenting though.